### DISCUSSION TITLE
Move to Python 3 only

### DISCUSSION THREAD

**Discussion by JohnVillalovos** [beaker-project/beaker#113] [Ideas]
I believe our plan is to move to Python 3 only in the future.

Would like to lay out our plan to move to Python 3.

**Comment by StykMartin** [comment]
Python 2 -> Python 6 (Server) -> Python 3 Flask
Python 2 -> Python 3 Client / Labcontroller / Common

Dropping dependencies from Client / Labcontroller / Common is quite easy, however, moving forward to Python 3 in Server is quite complicated. The whole core of the Server is based on the libraries that are not supported in Python 3. Therefore it is nice to keep Python 2 there for a while removing dead dependencies from the code
